include/solana.hpp

```diff
@@ -542,6 +543,12 @@ class Connection {
   uint64_t getBlockHeight(
       const Commitment &commitment = Commitment::FINALIZED) const;
 
+  /**
+   * Returns of the current Transaction has been confirmed or not
+   */
+  bool confirmTransaction(std::string transactionSignature,
+                          Commitment confirmLevel, uint8_t timeout=200) const;
```

Comment:
timout should be renamed with retries.

Reply:
I was think like it would be the amount of time the confirmTransaction would wait before returning a failure?